go.sum

```diff
@@ -10,14 +12,22 @@ github.com/richardlehane/webarchive v1.0.0 h1:RnmjrTFzNoenSypXrIFW0dvw8Isytv5z0N
 github.com/richardlehane/webarchive v1.0.0/go.mod h1:R+J0ocBfFb0A3vzwXZwrYoPe69PJLEejLzFL5ceDroM=
 github.com/richardlehane/xmldetect v1.0.2 h1:/3ooFuJwtgpMMe14/7m8a/JIvECMx6SpsPcDRiNyR8o=
 github.com/richardlehane/xmldetect v1.0.2/go.mod h1:Zp1lhTLRJa2p2QKA4jOruVQYc0NFQDO0YUz3k/k6JcE=
-github.com/ross-spencer/spargo v0.0.0-20200323024642-38971d4365a7 h1:G50l+RXrUyL5DE+Mj1+OOJgOR+hq8Ghf/ozx3FFcffQ=
-github.com/ross-spencer/spargo v0.0.0-20200323024642-38971d4365a7/go.mod h1:5mytCwysAzmwG9GJTFD7GR8+ZrhStjTOe3krU9Rlm8c=
+github.com/ross-spencer/spargo v0.0.0-20210125002137-2857c174aae4 h1:mlqd1wEbwvkjvwy9UjKzeZh3VybQZoOIMIiB5m53IIo=
```

Comment:
From `wikiprov` even though the package has its own `spargo` it implements `spargo` by exporting `spargo` structs and constants. The `wikiprov` package should be entirely compatible with aspects of `spargo`.